index.bs

```diff
@@ -1311,7 +1311,29 @@ The {{XRInputSource/gamepad}}'s {{Gamepad/id}} also enforces additional behavior
   - If the input device cannot be reliably identified the {{XRInputSource/gamepad}}'s {{Gamepad/id}} MUST be <code>"unknown"</code>.
   - If the UA masks the input device type for any reason the {{XRInputSource/gamepad}}'s {{Gamepad/id}} MUST be <code>"unknown"</code>
 
-ISSUE: <a href="https://github.com/immersive-web/webxr/issues/550">GitHub #550</a> - Additional restrictions, still under discussion, will be applied to the formatting of the {{XRInputSource/gamepad}}'s {{Gamepad/id}} attribute. Until those restrictions have been identified, all {{XRInputSource/gamepad}} {{Gamepad/id}}'s should default to <code>"unknown"</code>.
+When a {{XRInputSource/gamepad}}'s {{Gamepad/id}} is not "unknown", it MUST conform to the following format:
```

Comment:
For whatever reason I thought the "it MUST conform" was applying to the gamepad not the id and I was super confused for a moment...